### PR TITLE
New Package: go-cqhttp-1.0.0

### PR DESCRIPTION
**Pull request by CoolPlayLin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110406&drop=dogfoodAlpha

### CODE REVIEW

**Comment by CoolPlayLin** [comment]
@jedieaston

**Comment by stephengillie** [comment]
This package appears to have a dependency on `GoLang.Go`:
image

(Internet translation)[Do not run this program directly by double-clicking, which will lead to some unexpected consequences. Please run ./go cqhttp.exe in the shell, click OK to release the safe startup script, click cancel to close the program]

**Comment by CoolPlayLin** [comment]
The program is a CLI, It must run with shell@stephengillie

**Comment by CoolPlayLin** [comment]
It isn't dependency error,@stephengillie just run it with illegal way

**Comment by CoolPlayLin** [comment]
@Trenly

File: manifests/m/Mrs4s/go-cqhttp/1.0.0/Mrs4s.go-cqhttp.locale.zh-CN.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Mrs4s.go-cqhttp
+PackageVersion: 1.0.0
+PackageLocale: zh-CN
+Publisher: Mrs4s
+PublisherUrl: https://github.com/Mrs4s
+PublisherSupportUrl: https://github.com/Mrs4s/go-cqhttp
```
Comment (Trenly) [inline manifests/m/Mrs4s/go-cqhttp/1.0.0/Mrs4s.go-cqhttp.locale.zh-CN.yaml:9]:
```suggestion
PublisherSupportUrl: https://github.com/Mrs4s/go-cqhttp/issues
```

File: manifests/m/Mrs4s/go-cqhttp/1.0.0/Mrs4s.go-cqhttp.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+PrivacyUrl: https://github.com/Mrs4s/go-cqhttp
```
Comment (Trenly) [inline manifests/m/Mrs4s/go-cqhttp/1.0.0/Mrs4s.go-cqhttp.locale.zh-CN.yaml:10]:
```suggestion
# PrivacyUrl:
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by CoolPlayLin** [comment]
@Trenly

**Comment by stephengillie** [comment]
> It isn't dependency error,@stephengillie just run it with illegal way

Ah, this works:
```
未找到配置文件，正在为您生成配置文件中！
请选择你需要的通信方式:
> 0: HTTP通信
> 1: 云函数服务
> 2: 正向 Websocket 通信
> 3: 反向 Websocket 通信
请输入你需要的编号(0-9)，可输入多个，同一编号也可输入多个(如: 233)
您的选择是:exit
默认配置文件已生成，请修改 config.yml 后重新启动!
quit
```

(By "illegal way" I'm guessing you mean "not recommended way". 😄)

**Comment by CoolPlayLin** [comment]
It's working.

**Comment by CoolPlayLin** [comment]
The program needs a config file to run, When the cfg doesn't exist, It will guide u to generate one

@stephengillie

**Review by stephengillie — APPROVED** [review]

**Comment by CoolPlayLin** [comment]
@stephengillie What's your online time?

**Comment by CoolPlayLin** [comment]
Hi @stephengillie,
What's your local timezone?

**Comment by stephengillie** [comment]
> Hi @stephengillie, What's your local timezone?

I live in PST and work in PDT currently.

**Comment by CoolPlayLin** [comment]
> > Hi @stephengillie, What's your local timezone?
> 
> I live in PST and work in PDT currently. 

Sorry, I am new to English. I cannot understand these abbreviations

**Pull request merged**